.gitignore

```diff
@@ -3,6 +3,7 @@ dist
 node_modules
 .nyc_output
 .DS_Store
+tsconfig.tsbuildinfo
```

Comment:
I not think this or `.rollup.cache/` need to be here. Have noticed them created when the rollup config is not exactly as needed, but once finished, those were not created any longer.